### ISSUE TITLE
Rails/HttpStatus doesn't detect status key properly

### ISSUE THREAD

**Issue by btoconnor** [bbatsov/rubocop#5738]
I recently upgraded to 0.54.0 and noticed that the new Rails/HttpStatus cop was not picking up the following issue:

```ruby
render status: 201, json: {}
```

After spending an hour or so ensuring that everything was setup properly, I went and tried the following:
```ruby
render json: {}, status: 201
```
and that code correctly tripped the new HttpStatus error.

--------

## Expected behavior

I expected rubocop to pick up the status parameter to render regardless of position

## Actual behavior

Rubocop did not trigger a linting error.  It correctly triggers a linting issue when the code is formatted as
```ruby
render json: {}, status: 201
```

Screenshot from my editor with rubocop integrated:
![image](https://user-images.githubusercontent.com/81829/38140880-e28bed28-3403-11e8-956f-960e27adcfbe.png)

## Steps to reproduce the problem

In a controller, write
```ruby
render status: 201, json: {}
```
with `Rails/HttpStatus` enabled, and `EnforcedStyle` set to `symbolic` (the default)

## RuboCop version

Include the output of `rubocop -V`. Here's an example:

```
$ rubocop -V
0.54.0 (using Parser 2.5.0.5, running on ruby 2.4.2 x86_64-linux)
```
